README.md

```diff
@@ -1,12 +1,78 @@
 # Kinase
-Kinase (pronounced ki·nase) is an extension for Google Chrome allowing users to map web content to a list of annotations.
+Kinase (pronounced ki·nase) is a pluggable browser extension allowing you to label content on the web and use it for a wide variety of machine learning and data collection applications. By providing a set of `annotations` and their corresponding schemas, you can use Kinase to select the content best representing these annotations.
 
-## Contexts
-In Kinase, content mapped to the provided annotations is grouped together in a context. This context is keyed by an arbitrary string in the redux state, and might represent a specific user tagging content in the extension or a project that content is being tagged for.
 
-To switch contexts from another extension or webpage (e.g., if a user first reads an instruction page, you can set the project context from there), you can send a message to the extension with:
+## Concepts
+### Annotations
+Annotations represent the labels which you’d like to select content for. An annotation is represented by a string identifier (e.g., `products`) and a schema containing the fields and their types that can be mapped to the annotation (e.g., `name`, `description`, `photo`). The schema can optionally specify the `multiple` flag to indicate that multiple groupings of these fields can be mapped to the annotation.
+
+NB: Currently only `text`, `rich-text`, and `image` fields are supported. Help us out and add more!
```

Comment:
The currently supported field types are `text`, `rich-text`, and `image`.  Help us out and add more!